### DISCUSSION TITLE
ChatGPT Plus

### DISCUSSION THREAD

**Discussion by bandaid** [xtekky/gpt4free#367] [General]
I am a paying user of ChatGPT Plus and have to say that ANY and ALL AI models ought to be FREE to ALL. There ought to be absolutely no fee for its use or any other derivative. Especially with the number of investor schmucks who have thrown money at OpenAI looking for a massive payday.

**Comment by Seredeep** [comment]
I mean... These services cost A LOT to run.
Just talking about running any model that comes close to GPT-4 means at least 24GB of VRAM...
IF (and that's a big if) models manage to reduce in computing power required to run them, we might see some actual democratization of this technology.

Otherwise... It's just not viable.
Sadly i think this project won't last long because it's unsustainable

**Comment by hrizal** [comment]
Hopefully in the future there will be available OpenLLM as well as processing applications that use combined machines that are openly connected to each other to be used together.

**Reply by Seredeep** [reply]
something like internet computer?
there still is some cost to that. Unless the models actually reduce their footprint to be able to be ran in any computer you won't be getting anything just for free. Even then you're just distributing the cost of running it to every user. They pay their electric bill :)

Bing chat works "for free" but you're actually feeding it data so not free at all. Nothing is free in this life 😢
